src/java/picard/sam/markduplicates/EstimateLibraryComplexity.java

```diff
@@ -127,13 +127,21 @@
             "group size would be approximately 10 reads.")
     public int MAX_GROUP_RATIO = 500;
 
+    @Option(doc = "Barcode SAM tag (ex. BC)", optional=true)
```

Comment:
I see now. Yes using the BC and BX makes sense. Except if you have read errors in the sample index. You could easily do sample index correction I guess. We use the BX tag not the RX tag for our duplicate marking. And in the case of no BX tag we fall back to just positions. There should always be an RX tag for 10X data but a read without a BX tag means that we could not confidently error correct the raw barcode to a whitelist barcode.